docs/explanation/stable-release-updates.rst

```diff
@@ -1,2 +1,257 @@
+   number in the changelog using the ``LP: #NNNNNN`` format, and only reference public bugs.
+7. Once the :term:`Ubuntu SRU Team` reviews and accepts your upload, test the binaries
+   in the :term:`Ubuntu Archive` and follow up in the bug report with your verification
+   results. The :term:`Ubuntu SRU Team` will evaluate the testing feedback and move the
+   package into :ref:`updates <ArchivePockets_Updates>` after it passes a minimum aging
+   period of 7 days.
+8. Subscribe to the bugmail of the package in :term:`Launchpad`, and monitor
+   :term:`Launchpad` for bug reports relating to the update for at least one week. If
+   you notice and confirm any regression, document it in a bug report marked with an
+   ``Importance: critical`` label.
+
+SRU phasing
+-----------
+
+Once a package is released to :ref:`updates <ArchivePockets_Updates>`,
+the update is then phased so that the update is gradually made available to expanding
+subsets of Ubuntu users.
+
+Initially, the ``Phased-Update-Percentage`` is set to 10%, with a job running every 6
+hours to monitor for :ref:`regressions <Regressions>`. If no issues are detected, the
+update percentage increments by 10% until it reaches 100%. So an update will become
+fully phased after 54 hours. If a regression is found, the update is halted and the
+``Phased-Update-Percentage`` is set to 0%. This will then cause supported package
+managers not to install the update.
+
+Investigating halted phased updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To investigate why phasing stopped, use the phased updates report.
+
+When investigating an increased rate of crashes, focus on the crashes with the highest
+number of occurrences. Examine the occurrences table to determine if these crashes are
+happening more frequently with the updated version of the package. If they are,
+investigate the cause and address the crash in a follow-up :term:`SRU`. If not,
+`contact the Ubuntu SRU Team`_ about overriding the crash report.
+
+For new errors, verify that they're indeed new by reviewing the versions table and checking
+the ``Traceback`` or ``Stacktrace`` to determine if the error originates from the updated
+package or an underlying library. If you believe the error wasn't caused by the update,
+you can  `contact the Ubuntu SRU Team`_ to override the crash.
+
+Overriding halted phased updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Overriding halted phasing is similar to handling :term:`autopkgtest` failures.The phased
+update machinery uses a file named ``phased-update-overrides.txt``, a simple CSV file
+containing lines of the form ``source package``, ``version``, and ``$THING_TO_IGNORE``.
+
+``$THING_TO_IGNORE`` can either be an ``errors.ubuntu.com`` problem URL to ignore or
+``increased-rate``.
+
+.. _Verification:
+
+Verification
+------------
+
+:term:`SRU` verification should be done in a software environment that closely resembles
```

Comment:
nitpick: This page already established the acronym. There is no value in linking to the glossary.

```suggestion
SRU verification should be done in a software environment that closely resembles
```